mappings/net/minecraft/datafixer/fix/BlockEntityCustomNameToTextFix.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_3567 net/minecraft/datafixer/fix/BlockEntityCustomNameToComponentFix
+CLASS net/minecraft/class_3567 net/minecraft/datafixer/fix/BlockEntityCustomNameToTextFix
```

Comment:
This name is given by a string in `makeRule`, so it should be left the way it is.

Reply:
My reason for the change is that we don't use Component as a name anymore

Reply:
I support i509VCB here.